src/components/Navigation/Navigation.tsx

```diff
@@ -55,6 +55,9 @@ function Navigation() {
 						<PrivateNavLinkItem authorized={isAdmin} href="/admin/dashboard">
 							Admin
 						</PrivateNavLinkItem>
+						<NavLinkItem href="https://forms.gle/yM8Revi2NLHGNKs17">
```

Comment:
And we could now link the source URL from the navigation i.e. `/report` instead of the form.

Reply:
Updated in `532f374`